WordPress/src/main/res/color/on_surface_medium_secondary_selector.xml

```diff
@@ -2,8 +2,8 @@
 
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <item android:alpha="@dimen/disabled_alpha" android:color="?attr/colorOnSurface" android:state_enabled="false" android:state_selected="false" />
-    <item android:alpha="@dimen/disabled_alpha" android:color="?attr/colorOnSurface" android:state_enabled="false" android:state_selected="true" />
+    <item android:alpha="@dimen/reader_comments_like_count_disabled_alpha" android:color="?attr/colorOnSurface" android:state_enabled="false" android:state_selected="false" />
+    <item android:alpha="@dimen/reader_comments_like_count_disabled_alpha" android:color="?attr/colorOnSurface" android:state_enabled="false" android:state_selected="true" />
```

Comment:
💡 Suggestion: since we are already using the emphasis `medium` alpha from the material lib (line 8) why don't we also use the emphasis `disabled` alpha for this one, which is `material_emphasis_disabled` and has value `0.38`. 

That's the one used by default in material component texts and outlines so I think it would make sense to use that, and it's pretty close to what you're proposing here.

Reply:
Good idea, thanks! ✅